src/cljs/xt_play/view.cljs

```diff
@@ -26,6 +25,13 @@
 
 (defn- spinner [] [:div "Loading..."]) ;; todo spinners spin
 
+(defn- editor-update-opts [id source]
```

Comment:
This seems like an over-generalisation to me, not hung up on it though

Reply:
I'm going to leave this as is - reasoning: if we have a single place where we determine the editor behaviour it means any updates / fixes to come in the future will impact all.
If we were to move this logic to the editor ns, I fear it would have too much context about what's being updated :)